### PR TITLE
ref(integration-platform): Link to user payload for webhooks

### PR DESCRIPTION
**Pull request by MeredithAnya**
Screen Shot 2019-10-22 at 10 38 37 AM

### CODE REVIEW

File: src/collections/_documentation/workflow/integrations/integration-platform/webhooks.md [hunk 1]
```diff
@@ -531,6 +531,15 @@ The `error.created` resource subscription is only available for Business plans a
 - type: string
 - description: the api url for the associated issue
 
+{% capture markdown_content %}
+If you've set up [capturing the user](https://docs.sentry.io/enriching-error-data/context/?platform=python#capturing-the-user),
+you can find the user attributes under `data['error']['user']`.
+{% endcapture %}
+{% include components/alert.html
+  title="Note"
+  content=markdown_content
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/integrations/integration-platform/webhooks.md]:
```suggestion
  content=markdown_content
  level="info"
```

Comment (MeredithAnya) [reply src/collections/_documentation/workflow/integrations/integration-platform/webhooks.md]: are the notes supposed to look different with the different levels? testing locally it doesn't seem to change but wasn't sure if I'm missing something

Comment (MeredithAnya) [reply src/collections/_documentation/workflow/integrations/integration-platform/webhooks.md]: oh wait it does - just hadn't saved the file 😅

**Comment by MeredithAnya** [comment]
Screen Shot 2019-10-22 at 12 45 08 PM

**Pull request merged**